source/api/commands/type.md

```diff
@@ -48,7 +47,7 @@ Sequence | Notes
 `{selectall}` | Selects all text by creating a `selection range`
 `{uparrow}` | Fires up event but does **not** move the cursor
 
-Text passed to `.type()` may also include any of the these modifier character sequences:
+Text passed to `.type()` may also include any of these modifier character sequences:
```

Comment:
`Minor` `Typo`

There's a few such occurrences of these double words. I'll submit a distinct PR in a bit to address those so that this PR remains neat.